packages/utils/src/dappkit.ts

```diff
@@ -150,27 +152,40 @@ export function serializeDappKitRequestDeeplink(request: DappKitRequest) {
 
 // TODO: parsing query params yields broad types
 // once interface stabilizes, properly type the parsing
-export function parseDappkitResponseDepplink(url: string): DappKitResponse {
+export function parseDappkitResponseDepplink(url: string): [DappKitResponse, string] {
   const rawParams = parse(url, true)
   if (rawParams.query.type === undefined) {
     throw new Error('Invalid Deeplink: does not contain type:' + url)
   }
 
+  if (rawParams.query.requestId === undefined) {
+    throw new Error('Invalid Deeplink: does not contain requestId')
+  }
+
+  // @ts-ignore
+  const requestId: string = rawParams.query.requestId
```

Comment:
think you can do `as string` at the end to remove the tsignore

Reply:
another puppy dies.
What's the type of `rawParams.query`. Why is this failing?
`as string` works, but it also tells the compiler "i know this is a string"... 
it's better than using `ts-ignore` but i think here we can go without it

Reply:
`rawParams.query` is  `string | string[]`

Reply:
then `as string` or wrap it with `if (typeof rawParams.query === 'string')`
That's typescript type guards

Reply:
yeah i think if you change to throw an error if the typeof is wrong, you should be okay